### DISCUSSION TITLE
Online meetup?

### DISCUSSION THREAD

**Discussion by beshur** [crowdin/crowdin-cli#469] [Ideas]
Hello guys.

What do you think of hosting a small workshop or just an online meetup on CrowdIn tools?

We kind of struggle to get it right.
We set up a workflow, but it's not ideal, still includes manual work on developer side.
I want to share some ideas/code snippets that we have and see what other people doing with your service.

**Comment by dies** [comment]
Hi Alex

I would love to join! can you please ping me serhiy at crowdin com

**Comment by beshur** [comment]
Let me prepare some things, and a date for the event. Will you be able to notify other developer via your channels, @dies?

**Comment by dies** [comment]
100% @beshur 

thank you!

**Reply by beshur** [reply]
Cool, thanks.
